### ISSUE TITLE
Update ActiveObs to compare MOP's DB with Observe Portal

### ISSUE THREAD

**Issue by rachel3834** [LCOGT/mop#102]

**Comment by rachel3834** [comment]
The current ActiveObs tab queries MOPs DB for observation requests from the last 24 hours.  
A more useful display would include all currently-pending observations. 

It would also be useful to compare the observations MOP knows about with those from the LCO Observe Portal

**Comment by rachel3834** [comment]
Added a new table to the ActiveObs display to present the user with a display of the currently-pending observations read from the LCO Observe Portal as a sanity check to compare with what MOP thinks it has requested.  

This does not yet function in an intuitive way.  Firstly, the ActiveObs page is not displaying observations that were requested over the last week, since the period to be display was extended from 24hrs.  This is because the view is searching for observation records by scheduled_start and end times, which are not being set when obs_control saves the observation_records.  

- [ ] Update obs_control's observation submission to set the scheduled start and end times when run_TAP requests observations. 
- [ ] Add management command to back-fill this information for the rest of the observation records
- [ ] Update the table of observations from the LCO Portal to display ALL observations within the look-back period, indicating the status of each request.

**Comment by rachel3834** [comment]
run_TAP will now set the scheduled start and end times whenever it requests new observations, and I have developed and run a management command on the rest of MOP's existing ObservationRecords to back-fill this information from the stored JSON parameters.  This means that the ActiveObs table can accurately select ObservationRecords to display within the last week. 

I've also updated the table of observations known to the LCO Portal, to include status observations, and to include observations regardless of scheduling status.
